static/js/flow/enrollmentTypes.js

```diff
@@ -13,4 +13,5 @@ export type ProgramEnrollmentsState = {
   getStatus?: string,
   getErrorInfo?: APIErrorInfo,
   postStatus?: string,
+  coursePostStatus?: string,
```

Comment:
on first look it seems appropriate to add the state information for course enrollment results to the program enrollments state (they're both enrollments after all), but this is kinda awkward. course enrollments are, after all, very different from program enrollments, and `ProgramEnrollmentsState` was created to reflect information about API calls to program enrollment endpoints. we're just sort of gluing this extra piece of information to this unrelated state when we need to. i guess it's not crucial to create some new state type to store this, and i'm not sure that storing it in DashboardState is any cleaner - if you think of something easy and more sensible to do with this, that'd be great, but i won't hold up the PR for it. if you decide to keep this as part of `ProgramEnrollmentsState`, please change the key to `courseEnrollAddStatus` or something like that. `coursePostStatus` is misleading